rpc/server_auth.go

```diff
@@ -374,12 +374,11 @@ func (ss *simpleServer) ensureAuthed(ctx context.Context) (context.Context, erro
 				"error", err,
 				"empty expected audience", len(ss.authAudience) == 0,
 				"expected audience list", audienceList)
-			//return nil, status.Error(codes.Unauthenticated, "invalid audience")
```

Comment:
This was a mistake. Eliot tried commenting it out but it seems like the build didn't like that.